config/env/default.js

```diff
@@ -8,13 +8,20 @@ module.exports = {
     googleAnalyticsTrackingID: process.env.GOOGLE_ANALYTICS_TRACKING_ID || 'GOOGLE_ANALYTICS_TRACKING_ID'
   },
   port: process.env.PORT || 3000,
+  token: {
```

Comment:
This was necessary to make the build work, and is what mem-admin does.

Basically, this token stuff is part of the regular build process, but was only specified in some of the configurations.  Moving it here since its needed regardless of which NODE_ENV you use.